bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt

```diff
@@ -11,6 +11,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: <%= RUBY_VERSION %>
+        bundler-cache: true
```

Comment:
Yes! But then the `gem install` && `bundle install` lines below can be removed :)

Reply:
Oh yes, updated.